src/pages/reference/handlers/json-schema.md

```diff
@@ -303,13 +303,16 @@ In this example, we declare the `limit` parameter with a default value of `10` a
 -  `operations` - (required) Array of:
    -  `object`:
       -  `field` (type: `String`, required)
+         -  `operations.field` cannot contain hyphens.
```

Comment:
This should actually be `operations[0].field`. However, I think we can omit it since the object structure is already clear here.
```suggestion
         -  Cannot contain hyphens.
```